prisma/schema.prisma

```diff
@@ -385,6 +385,7 @@ model Transaction {
   @@index([paymentReference, paymentStatus])
   @@index([transactionId])
   @@index([type])
+  @@fulltext([paymentReference, senderIdentifier, receiverIdentifier])
```

Comment:
Include transactionId field. Which value is the receiverIdentifier representing?